validator/validators/feature/feature_validator.go

```diff
@@ -64,3 +105,87 @@ func (s *FeatureValidator) Cleanup() error {
 
 	return nil
 }
+
+func (s *FeatureValidator) ValidateLogs(logStream string, logLine string, numberOfLogLine int, startTime, endTime time.Time) error {
+	var (
+		logGroup = awsservice.GetInstanceId()
+	)
+	log.Printf("Start to validate log '%s' with number of logs lines %d within log group %s, log stream %s, start time %v and end time %v", logLine, numberOfLogLine, logGroup, logStream, startTime, endTime)
+	ok, err := awsservice.ValidateLogs(logGroup, logStream, &startTime, &endTime, func(logs []string) bool {
+		if len(logs) < 1 {
+			return false
+		}
+		actualNumberOfLogLines := 0
+		for _, l := range logs {
+			if strings.Contains(l, logLine) {
+				actualNumberOfLogLines += 1
+			}
+		}
+
+		return numberOfLogLine <= actualNumberOfLogLines
```

Comment:
Should this be a more strict `==` check? If we know exactly how many log lines were written to the file, we should be able to expect that exact amount, right?